bigbio/hub/hub_repos/samd/README.md

```diff
@@ -0,0 +1,37 @@
+
+---
+language: 
+- en
+bigbio_language: 
+- English
+license: UNKNOWN
+multilinguality: monolingual
+bigbio_license_shortname: UNKNOWN
+pretty_name: Sentiment Analysis for Medical Drugs (SAMD)
+homepage: https://www.kaggle.com/datasets/arbazkhan971/analyticvidhyadatasetsentiment
+bigbio_pubmed: False
+bigbio_public: False
+bigbio_tasks: 
+- SEMANTIC_SIMILARITY
```

Comment:
@mariosaenger I think the schema is alright, but maybe the task "TEXT_PAIRS_CLASSIFICATION" is more appropriate (as it is more generic)?

Reply:
See also: https://github.com/bigscience-workshop/biomedical/issues/682#issuecomment-1146717432

Reply:
Thanks for the hint. Revised it to TEXT_PAIRS_CLASSIFICATION.